src/Compatibility/Microsoft.DotNet.PackageValidation/ApiCompatRunner.cs

```diff
@@ -35,9 +35,7 @@ public void RunApiCompat()
         {
             foreach (var apicompatTuples in _queue.Distinct())
             {
-                // TODO: Add Assembly version check.
```

Comment:
I don't see the assembly check, are you removing this cause you opened the issues I guess?

Reply:
yep https://github.com/dotnet/sdk/issues/17507